src/components/People/People.tsx

```diff
@@ -0,0 +1,90 @@
+import { Link, useParams } from 'react-router-dom';
+import data from '../../../public/api/people.json';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+const getSlugByName = (name: string) => {
+  const person = data.find(p => p.name === name);
+
+  return person ? person.slug : null;
+};
+
+interface PeopleProps {
+  people: Person[];
+}
+
+export const People = ({ people }: PeopleProps) => {
+  const { slug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => (
+          <tr
+            key={person.slug}
+            className={classNames({
+              'has-background-warning': person.slug === slug,
```

Comment:
The class `has-background-warning` is applied when `person.slug === slug`. Ensure that `slug` is correctly obtained from the URL parameters to avoid unexpected behavior.